spec/jobs/solidus_bolt/shipment_tracking_job_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::ShipmentTrackingJob do
+  subject(:shipment_tracking_job) {
```

Comment:
since it spans multiple lines prefer a do ... end

Reply:
I don't think this statement is large enough to warrant a `do...end` block for itself.